parcels/numba/README.md

```diff
@@ -0,0 +1,49 @@
+# Numba for Parcels, a report
+
+This is a report for the Numba-Parcels project of the ITS-Parcels team. As a summary, we have investigated the feasibility of implementing a parcels version that runs on Numba, instead of JIT/Scipy. In the end we find that at this stage it is probably possible, but would still require a lot of work, while being much slower than JIT and less flexible than Scipy, with higher maintanance. The main difficulties arise because of Parcels design philosphy of allowing users a lot of leeway in defining types and variables in their simulation. While this of course is one of the main strengths of Parcels as a library, it does pose challenges when working with Numba.
+
+## Benchmarks
+
+The benchmark script is located under `parcels/examples/benchmark_stommel.py`, which uses the parcels `stommel` benchmark script `parcels/examples/example_stommel.py`.
+
+The script runs for 600 simulation days with a delta of 1 hour, which amounts to 14,400 simulation steps. This is done for a recent version of the main parcels branch for `scipy` and `JIT` (25 december 2021). The `numba` results are created with this branch. The final positions were validated against the main parcels branch.
```

Comment:
Link to the actual commit ID here, instead of the date?

Reply:
Hmmm, I don't think I have a commit anymore..